mysql-test/suite/innodb/t/tablespace_encrypt_9.test

```diff
@@ -149,7 +149,6 @@ CREATE TABLESPACE temp_ts ADD DATAFILE 'temp_ts.ibd';
 --echo # connection con3
 connect (con3,localhost,root,,);
 connection con3;
-SELECT connection_id();
```

Comment:
Is this related to telemetry or was this some other issue?

Reply:
Having connection_id recorderd in the result seems to be not good idea, as it is a random number in general. The reason was internal connection made by telemetry component.